build.rs

```diff
@@ -256,7 +256,7 @@ const DATA: &'static [(&'static str, &'static [&'static str], &'static [&'static
     ("winineti", &["minwindef"], &[]),
     ("winioctl", &["devpropdef", "minwindef", "winnt"], &[]),
     ("winnetwk", &["basetsd", "minwindef", "windef", "winerror", "winnt"], &["mpr"]),
-    ("winnls", &["basetsd", "guiddef", "minwindef", "winnt"], &[]),
+    ("winnls", &["basetsd", "guiddef", "minwinbase", "minwindef", "winnt"], &[]),
```

Comment:
You need to add `"kernel32"` to the second array, because this module is linking in symbols from `kernel32.lib`.